pkgs/top-level/python-packages.nix

```diff
@@ -12500,6 +12500,28 @@ in {
     };
   };
 
+  hkdf = buildPythonPackage rec {
+    name = "hkdf-${version}";
+    version = "0.0.3";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/h/hkdf/${name}.tar.gz";
+      sha256 = "1jhxk5vhxmxxjp3zj526ry521v9inzzl8jqaaf0ma65w6k332ak2";
+    };
+
+    buildInputs = with self; [ nose ];
+
+    checkPhase = ''
+      nosetests
+    '';
```

Comment:
@FRidh hkdf is actually [using nose](https://github.com/casebeer/python-hkdf/blob/master/tox.ini), shouldn't we use it too?

The only issue is that this seems to run 0 tests for some reason..